content/docs/guides/scim/okta.md

```diff
@@ -0,0 +1,136 @@
+
+    * givenName
+    * familyName
+    * displayName
+    * email
+
+5. Click **Save Mappings**.
+6. Click **Apply Updates**.
+
+## Setting up Group Provisioning
+
+The Pulumi console supports the provisioning of teams within your organization using SCIM. This is done by mapping the groups you have created using SCIM to create teams within your organization in the Pulumi console. Setting this up allows you to manage your teams' memberships solely in Okta.
+
+To set this up, you need to enable Push Groups as a supported provisioning action under the **Provisioning** settings and then specify which groups you would like to push. To do that, performing the following steps:
+
+1. Navigate to the application that you created above by clicking the on the **Applications** tab and then selecting the application.
+
+    ![Okta application selection.](/images/docs/guides/scim/okta-application-selection-scim.png)
+
+2. Click on **Provisioning** tab, then click on the integrations menu option located on the side navigation.
+3. Check the **Push Groups** box under **Supported provisioning actions**.
+4. Select the **Push Groups** tab.
+5. Select the **Push Groups** drop down menu. Here you will have the option to find groups either by name or rule.
+
+    ![Push Groups drop down menu in Okta](/images/docs/guides/scim/okta-pushgroups-menu-scim.png)
+
+6. Select **Find groups by name**. This will take you to a screen where you can search groups by the group name. Once you find the group(s) that you are looking for, click **Save**.
+
+    You will notice there is a checkbox labeled **Push group memberships immediately**. This will also add all the members that are part of your group to the Pulumi team after your selection has been saved.
+
+7. After clicking **Save**, you should see that your group was pushed and it should say **Active**.
+
+    Note: if there are members in a group that are not yet assigned to the Pulumi Console application in Okta, they will not be added to the team in the Pulumi Console. Ensure that all members in the group have been assigned to the application before pushing the group.
+
+## Verifying Group Provisioning
+
+To confirm that the groups were provisioned correctly, log into the Pulumi Console and select the **TEAMS** tab where you should see the team listed.
+
+![Pulumi console showing current teams.](/images/docs/guides/scim/console.png)
+
+Click on that group and verify its membership. Note that the team’s membership cannot be altered within the Pulumi console. If any membership changes are needed, they must be done within Okta. This allows the membership to be managed using Okta so your teams on the Pulumi side will always mirror the groups you have configured there.
+
+## Known Limitations
+
+Here’s a list of the features that are not currently supported:
+
+* Secondary emails
+* Password sync
+* Bulk importing
+* Subgroups (groups composed of other groups, only groups composed of users are supported)
```

Comment:
I recently discovered that Okta does not support subgroups either so I think we can remove this item since this doc is Okta specific.